cms/CONTENTFUL.md

```diff
@@ -223,3 +223,47 @@ Information for onboarding content editors:
 - `API`: Application Programming Interface.
 - `Delivery API`: The mechanism that returns published content.
 - `Preview API`: The mechanism that returns both published and draft content.
+
+
+## Question types
```

Comment:
@martikat how does this read?

Reply:
This reads well from my perspective 👍  I think you have covered everything in enough detail here